mbed-util/FunctionPointerBase.h

```diff
@@ -31,6 +31,14 @@ class FunctionPointerBase {
         return (_membercaller != NULL) && (_object != NULL);
     }
 
+    bool operator==(const FunctionPointerBase& other) {
```

Comment:
this can be a const method. i.e. `bool operator== (const FunctionPointerBase & other) const {`